wechaty-puppet-hostie/src/main/kotlin/io/github/wechaty/grpc/GrpcPuppet.kt

```diff
@@ -39,7 +40,7 @@ class GrpcPuppet(puppetOptions: PuppetOptions) : Puppet(puppetOptions) {
 
 //    private var finishLatch:CountDownLatch? = null
 
-    private fun discoverHostieIpa(): String {
+    private fun discoverHostieIp():Pair<String,String>{
```

Comment:
I think the JSON data is `number` instead of `string`.

I hope it can be converted automatically.

Reply:
NumberUtils.toInt(discoverHostieIp.second) can convert it to Int